pulser/simresults.py

```diff
@@ -63,6 +68,8 @@ def expect(self, obs_list):
 
         qobj_list = []
         for obs in obs_list:
+            if not hasattr(obs, "shape"):
```

Comment:
I would have personnally checked for the possible types of obs.

Reply:
Made this check explicit for `qutip.Qobj`and `numpy.ndarray`